src/web/server/htmlTemplate.ts

```diff
@@ -203,8 +204,29 @@ export const htmlTemplate = ({
                     <img src="https://sb.scorecardresearch.com/p?c1=2&c2=6035250&cv=2.0&cj=1&cs_ucfr=0&comscorekw=${keywords}" />
                 </noscript>
                 ${[...priorityScriptTags].join('\n')}
-                <style class="webfont">${getFontsCss()}${resetCSS}${css}</style>
-
+                <style data-auto-foft-fonts>${getFontsCss()}</style>
+                <script>
+                // mark: PswXqO - keep these in sync with preloads
+                window.autoFoft = {
+                    isCritical: function (font) {
+                        switch (font.family) {
+                            case 'GuardianTextEgyptian':
+                            case 'Guardian Text Egyptian Web':
+                            case 'GuardianTextSans':
+                            case 'Guardian Text Sans Web':
+                                return font.weight === 'normal' || font.weight === '400';
```

Comment:
should this be `font.style === 'normal'`?

Reply:
ooh yeah good spot